qa/qa_scripts/openstack/files/cinder.template.conf

```diff
@@ -2810,7 +2810,7 @@ auth_uri = http://VARINET4ADDR:5000/v2.0
 # Complete admin Identity API endpoint. This should specify the unversioned
 # root endpoint e.g. https://localhost:35357/ (string value)
 #identity_uri = <None>
-identity_uri = http://VARINET4ADDR:35357
+identity_uri = http://VARINET4ADDR:5000
```

Comment:
I'm not really sure either it might be old stuff just laying around?

We could just ignore it yeah but since we remove that port for example the config would never work with the teuthology suite either way.